src/views/Target.vue

```diff
@@ -171,23 +171,8 @@ async function get_target_info() {
         import.meta.env.VITE_API_URL + `/target/pipelines/${sdss_id}?release=${rel}`
```

Comment:
will this still work if we remove the base urls here?

Reply:
Yes, it seems the URLs on `axiosInstance` need to have the initial / but they are concatenated with the base URL. It also accepts absolute URLs and the configuration (the `json-big` parsing) still gets applied.